.github/workflows/docs-noob-tester.lock.yml

```
@@ -0,0 +1,1296 @@
+#   - actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
+#   - actions/download-artifact@3e5f45b2cfb9172054b4087a40e8e0b5a5461e7c # v8.0.1
+#   - actions/github-script@373c709c69115d41ff229c7e5df9f8788daa9553 # v9
+#   - actions/github-script@3a2844b7e9c422d3c10d287c895573f7108da1b3 # v9
+#   - actions/upload-artifact@bbbca2ddaa5d8feaa63e36b76fdaad77386f024f # v7
+#   - github/gh-aw-actions/setup@2fe53acc038ba01c3bbdc767d4b25df31ca5bdfc # v0.68.1
+
+name: "Documentation Noob Tester"
+"on":
+  schedule:
+  - cron: "28 11 * * *"
+    # Friendly format: daily (scattered)
```

YAMLlint (1.38.0)

[warning] comment not indented like content

(comments-indentation)